es5-shim.js

```diff
@@ -176,6 +179,36 @@ var ES = {
     }
 };
 
+/**
+ * An implementation of `slice`. Primarily for use with `arguments`.
```

Comment:
oh oops, i'm referring to something in the es6-shim :-) https://github.com/paulmillr/es6-shim/blob/master/es6-shim.js#L1252-L1263